configuration/transformations.md

```diff
@@ -185,51 +185,32 @@ The script transformation is also available as profile. When acting as transform
 
 #### Script Transformation Profile Configurations
 
-| Parameter Name    | Description                                                                                |
-| ----------------- | ------------------------------------------------------------------------------------------ |
-| `toItemScript`    | The `script identifier` for performing transformations from the Thing handler to the item. |
-| `toHandlerScript` | The `script identifier` for performing transformations from the item to the Thing handler. |
+| Parameter Name          | Description                                                                                                     |
+|-------------------------|-----------------------------------------------------------------------------------------------------------------|
+| `toItemScript`          | The `script identifier` for performing transformations from the Thing handler to the item.                      |
+| `commandFromItemScript` | The `script identifier` for performing transformations of **commands** from the item to the Thing handler.      |
+| `stateFromItemScript`   | The `script identifier` for performing transformations of **state updates** from the item to the Thing handler. |
 
 Example usage in an `.items` file:
 
 ```xtend
-Number <itemName> { channel="<channelUID>"[profile="transform:JS", toItemScript="decode_json.js", toHandlerScript="encode_json.js" ] }
+Number <itemName> { channel="<channelUID>"[profile="transform:JS", toItemScript="decode_json.js", commandFromItemScript="encode_json.js" ] }
 ```
 
 Here, additional parameters can also be injected into the script using the URL style syntax, e.g.:
 
 ```xtend
-Number <itemName> { channel="<channelUID>"[profile="transform:RB", toItemScript="multiply.rb?factor=10", toHandlerScript="multiply.rb?factor=0.1" ] }
+Number <itemName> { channel="<channelUID>"[profile="transform:RB", toItemScript="multiply.rb?factor=10", commandFromItemScript="multiply.rb?factor=0.1" ] }
 ```
 
 Inline script is also supported in the profile syntax.
 
 ```xtend
-Number <itemName> { channel="<channelUID>"[profile="transform:RB", toItemScript="| input.to_f * 10", toHandlerScript="| input.to_f * 0.1" ] }
+Number <itemName> { channel="<channelUID>"[profile="transform:RB", toItemScript="| input.to_f * 10", commandFromItemScript="| input.to_f * 0.1" ] }
 ```
 
-More details regarding this and other Transformation services can be found in the individual transformation articles linked below.
```

Comment:
FYI this section was removed by the build script during build, so we can instead remove it from the source file and don't need to adjust the build script.